### PR TITLE
Start Jaeger to avoid errors related to opentracing/opentelemetry

### PR DESCRIPTION
**Pull request by zakkak**
Closes https://github.com/Karm/mandrel-integration-tests/issues/290

### CODE REVIEW

**Comment by Karm** [comment]
Hello @zakkak, we haven't had Jaeger there historically and we used to whitelist the connection error:

https://github.com/Karm/mandrel-integration-tests/blob/master/testsuite/src/it/java/org/graalvm/tests/integration/utils/WhitelistLogLines.java#L118

If I remember correctly, it was to avoid friction with handling yet another runtime source of entropy for the testsuite. It was also way before the time the container control flow and "cleaning" containers was introduced. So perhaps it could be fine now?  

 - lemme try on aarch64 Linux
 - and lemme try on mac as I am afraid the ssh tunnel hack might be needed? https://github.com/Karm/mandrel-integration-tests/blob/master/testsuite/src/it/java/org/graalvm/tests/integration/JFRTest.java#L383

**Comment by zakkak** [comment]
Hi @Karm any updates on this?

**Review by Karm — CHANGES_REQUESTED** [review]
@zakkak 



## Linux, aarch64, Podman:

First running:
```
$ mvn  clean verify -Ptestsuite -DexcludeTags=all -DincludeTags=perfcheck,reproducers -Dtest=PerfCheckTest#testQuarkusFullMicroProfile -Dquarkus.version=3.16.0
```

And then running:

```
$  mvn clean verify -Ptestsuite -Dtest=RuntimesSmokeTest#quarkusFullMicroProfile -Dquarkus.version=3.16.0 -Dquarkus.native.container-runtime=podman -Dpodman.with.sudo=false
```

gives:

```
Error: creating container storage: the container name "quarkus_jaeger" is already in use by b745adcccf87008e79f4d572b28320ddcdfa06968a8e43f63496fa44457270e1. You have to remove that container to be able to reuse that name: that name is already in use, or use --replace to instruct Podman to do so.
```

re-running 
```
$  mvn clean verify -Ptestsuite -Dtest=RuntimesSmokeTest#quarkusFullMicroProfile -Dquarkus.version=3.16.0 -Dquarkus.native.container-runtime=podman -Dpodman.with.sudo=false
```

Makes it all right, so IMHO the PerfCheck is just missing cleanup in finally?

## Mac

```
karm@mandrel-macos-aarch64-2 mandrel-integration-tests % mvn clean verify -Ptestsuite -Dtest=RuntimesSmokeTest#quarkusFullMicroProfile -Dquarkus.version=3.16.0 -Dquarkus.native.container-runtime=podman -Dpodman.with.sudo=false
```

Works fine  (with jaeger from Quay)

## Docker Hub

Dependency on dockerhub is IMHO a no-no, it will keep randomly collapsing on CI due to free tier rate limiting.  Does not show on my laptop wit Docker account but it does on random servers:

```
Error: initializing source docker://jaegertracing/all-in-one:latest: reading manifest latest in docker.io/jaegertracing/all-in-one: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```

Please, either find trustworthy jaeger on Quay or package our own since we really don't care about it being up to date in this scenario.  We don't hit this issue with e.g. hyperfoil as we use https://quay.io/repository/karmkarm/hyperfoil. Databases are also used from Quay...

e.g https://github.com/jaegertracing/jaeger/pull/2783  

so `quay.io/jaegertracing/all-in-one` works for me :+1: 


## TL;DR:
- pls add cleanup in PerfCheck test
- pls use Quay

**Comment by zakkak** [comment]
>   pls add cleanup in PerfCheck test
    pls use Quay

Done

**Review by Karm — APPROVED** [review]
Thx.
